egs/swbd/s5c/local/chain/run_tdnn_6z.sh

```diff
@@ -55,7 +55,7 @@ final_layer_normalize_target=0.5
 num_jobs_initial=3
 num_jobs_final=16
 minibatch_size=128
-relu_dim=576
+relu_dim=1024
```

Comment:
[6z.pdf](https://github.com/kaldi-asr/kaldi/files/187176/6z.pdf)

See the figure in the attached pdf. You can see that model 6z has a fork at the penultimate layer. There are 2 final layers one in the branch which goes to chain objfn and one in the branch which goes to xent objfn. You should count the parameters in the branch which goes to chain objn. You can read steps/nnet3/tdnn/make_configs.py to understand how the final " 0 " is interpreted by the python code to write out the model config.

We need to increase the relu_dim from 576 in 6z to 768 as the splicing indices in 6z splice 3 hidden outputs at the input of every tdnn layer while the splicing indices used in the local/nnet3/run_tdnn.sh splice only 2 hidden layer outputs at every tdnn layer.